index.rst

```diff
@@ -316,6 +316,17 @@ Documentation to be collected for the release notes in pipelines_lsst_io_ is:
   pipelines_lsst_io_ with the new download location of the ``newinstall.sh``
   script.
 
+Documenting Deprecations
+""""""""""""""""""""""""
+
+In order to add in the release note the deprecated methods and functions, search the codebase for ``@deprecated\(`` and ``deprecate_pybind11``. For example, in a MacOS console, the command **ack** can be used: ``ack -A 3 --python "@deprecated\(" stack/``. This will print out each occurrence of ``"@deprecated\("`` found and the following 3 lines.
```

Comment:
This looks like it's only searching for deprecations in Python code, rather than C++ — that seems strange. Am I missing something? It would also be nice to search for [deprecated config fields](https://developer.lsst.io/stack/deprecating-interfaces.html#config-deprecation).

Beyond that, I have a number of picky style points:

- Please _either_ use one line per sentence _or_ (for consistency with the rest of the file) a hard wrap at a specific column (looks like about 80, but I didn't check).
- Don't capitalize “macOS”. 
- The “Console” application on macOS is designed for viewing logs, not executing commands. Arguably that's a poor choice on Apple's part, but it makes the wording above confusing.
- macOS doesn't ship with `ack`, so the above will give you a “command not found”.
- (Given the three points above,) I wonder if the reference to macOS is actually helpful. Rather than referring to a particular OS and exotic tools, you could probably rewrite this text to refer to an invocation of `find ... -exec ...` “at a shell prompt”, which will work for everybody.
- Per the [style guide](https://developers.google.com/style/numbers#numbers-as-numerals_1), please spell out numbers less than 10 in words (“three”, and “two” below).

Reply:
I told @gcomoretto to use `ack` because it makes life really easy with regular expressions when searching solely for python files. Much easier than trying to get this to work in `find` and `egrep` to work. `ack` can be installed very easily. I agree that macOS is irrelevant here since any shell on linux or macOS is fine.

Reply:
Yeah, I guessed that — “just use ack” has @timj written all over it. :-)

I still think that telling people just to run some command, without even explaining to them that they probably don't have that command installed, isn't actually being helpful — however clever the command is.

Reply:
I added a footnote, clarifying that ack may need to be installed separately.